src/utils/api.js

```diff
@@ -18,19 +18,20 @@ import {
   TOP_NUMBER_OF_ITEMS_TO_DISPLAY,
 } from '../config/constants';
 
+const getActionDay = (action) => {
+  const dateKey = 'createdAt';
+  // createdAt should have the format "2020-12-31T23:59:59.999Z"
+  const dateObject = new Date(action[dateKey]);
+  // extract only the date information
+  const day = new Date(dateObject.toDateString());
+  return day;
+};
```

Comment:
@rayacers A bit late but could this also work ? Using de-structuring of the argument.
```ts
const getActionDay = ({ createdAt }) => {
  // createdAt should have the format "2020-12-31T23:59:59.999Z"
  const dateObject = new Date(createdAt);
  // extract only the date information
  const day = new Date(dateObject.toDateString());
  return day;
};
```